tools/clang/test/HLSL/workgraph/node_input_compatibility.hlsl

```diff
@@ -12,7 +12,7 @@ struct RECORD
 [Shader("node")]
 [NumThreads(1024,1,1)]
 [NodeDispatchGrid(64,1,1)]
-[NodeLaunch("BrOaDcasting")]                         // expected-note  {{Launch type defined here}}
```

Comment:
We should probably have tests that verify that we get an expected error with incorrect capitalizations like what was here originally.